src/internal/ButtonBase.js

```diff
@@ -121,6 +103,8 @@ export default class ButtonBase extends Component {
   button = null;
   keyboardFocusTimeout = undefined;
 
+  focus = () => this.button.focus();
```

Comment:
@oliviertassinari we could even remove this... although it's worth considering if it should be part of the public API so there is a react-based way to programatically focus our buttons. Unless you think that we should just recommend `findDOMNOde` for now. WDYT?